docs/getting_started/software_setup.rst

```diff
@@ -2,94 +2,69 @@
 Get started writing code for the MotorGo
 ========================================
 
-The MotorGo is programmable in the Arduino IDE or in PlatformIO. At it's core, the MotorGo is an ESP32-S3 microcontroller with built-in motor drivers, so it can be programmed just like any other ESP32. To make it easy to use the onboard features, we have published custom board definitions for the MotorGo and a MotorGo Mini driver library. The MotorGo Mini board defintions define the onboard led pins, I2C, and SPI pins so that you can use them in your code without having to look up the pin numbers. The MotorGo Mini driver library provides a simple interface for controlling the motors and reading the encoders. You only need to use the driver if you want to use the motors or encoders. If you want to use the onboard peripherals, you can use the MotorGo Mini board definitions without the driver library.
+The MotorGo is programmable in the Arduino IDE or in PlatformIO. At it's core, the MotorGo is an ESP32-S3 microcontroller with built-in motor drivers, so it can be programmed just like any other ESP32. Follow the steps below to get started writing code for the MotorGo in Arduino or PlatformIO.
 
 There are 3 tools you need to get started writing code for the MotorGo:
 
-* MotorGo Mini Board Definitions: Let's you program the MotorGo Mini in Arduino/PlatformIO
-* MotorGo Mini Driver: Provides functions for controlling the motors with the onboard motor controllers
-* MotorGo Mini GUI: Let's you tune your PID controllers over WiFi (optional)
+* ESP32 Boards: Provides board definitions for the MotorGo
+* MotorGo Mini Driver: Provides API/functions for controlling the motors with the onboard motor controllers
+* MotorGo Mini GUI: Lets you tune your PID controllers over WiFi (optional)
 
 Let's walk through the steps of setting each of these up.
 
-Setting up board definitions
-----------------------------
-
-Follow the steps below to set up the board definitions.
-
-First, install the ESP32 board definitions, if you have not already done so.
-
-**Arduino IDE 1.X/2.X**
+Arduino IDE
+-----------
+First, install the ESP32 board definitions, if you have not already done so. You need at least version 2.0.16 of the ESP32 board definitions for the MotorGo to work.
 
 * Go to `Tools > Board > Boards Manager...`
 * Search for `ESP32`
 * Install `esp32` by Espressif Systems
 
-**PlatformIO**
-
-* Navigate to PlatformIO Home
-* Go to `Platforms`
-* Search for `Espressif 32`
-* Click `Install`
-
-Next, follow the steps below to setup the MotorGo board definitions.
+Next, install the MotorGo Mini driver library.
 
-**Arduino IDE 1.X/2.X**
+* Download the libary by clicking `here <https://github.com/Every-Flavor-Robotics/motorgo-arduino/raw/main/motorgo-mini-driver.zip>`_
+* Go to `Sketch > Include Library > Add .ZIP Library...`
+* Select the downloaded file
```

Comment:
This is no longer the correct way to setup the driver. This needs to be changed to explain how to use the arduino library manager.

Reply:
This has been fixed